src/ext/java/org/opentripplanner/ext/fares/impl/HighestFareInFreeTransferWindowFareService.java

```diff
@@ -54,20 +55,21 @@ protected boolean populateFare(
     List<Leg> legs,
     Collection<FareRuleSet> fareRules
   ) {
-    float cost = 0;
-    float currentTransferWindowCost = 0;
+    var zero = Money.ofFractionalAmount(currency, 0);
```

Comment:
Can the `Money` class include a universal `zero` constant to reference here (and in the Atlanta fare class too)?

Reply:
I think I would not want a unversal zero as this sidesteps the whole currency check but I've introduced `Money.ZERO_DOLLARS` in https://github.com/opentripplanner/OpenTripPlanner/pull/5100/commits/633f42818749d08232eed4f5453715c0a102978c

Do you think this is good enough?

Reply:
Is `HighestFareInFreeTransferWindowFareService` supposed to handle any currency besides USD?

Reply:
In theory yes, although the logic is very US-specific so I would be suprised if it's going to be used outside the US.

Reply:
`Money.ZERO_DOLLARS` Is probably good enough for now. I suppose that later you could introduce
```
public static Money zero(currency) {
  return new Money(currency, 0);
}
```